devel/api.md

```diff
@@ -655,6 +661,58 @@ Type: `string`
 
 The interface name which current VLAN is based on.
 
+# VXLAN
+Besides basic interface properties, each VXLAN interface state also contains
+a dictionary saved in key `VXLAN.CONFIG_SUBTREE`.
+
+Example:
+
+```python
+{
+    Interface.KEY: [
+        Interface.NAME: "eth1.101",
+        Interface.TYPE: VXLAN.TYPE,
+        Interface.STATE: InterfaceState.UP
+        VXLAN.CONFIG_SUBTREE: {
+            VXLAN.ID: 101,
+            VXLAN.BASE_IFACE: 'eth1'
+            VXLAN.REMOTE: '192.168.3.3'
```

Comment:
There are commas missing in the example. Can you maybe pipe it through black to see that it is correct? Also the string constants use different quote notations.

Reply:
Also please use RFC5737 IP addresses: https://tools.ietf.org/html/rfc5737

Reply:
Didn't know about it, that's cool.